data/free worlds start.txt

```diff
@@ -1615,6 +1769,122 @@ mission "FW Pirates 4.1"
+			`	JJ shakes his head as he leans forward to respond. "Senator, this was never about being a hero to the galaxy to Captain <last>. This was about being a hero to those slaves. Many of them have been waiting for generations to escape to a chance at a better life?"`
+			`	The old senator blinks. "And you think Captain <last> can provide them a better life?"`
+			`	JJ shrugs. "They're free to find it themselves, just as we are. The free slaves seem determined to become their own heroes from now on, but I would think that the Senate and Council of the Free Worlds should be happy to help them find their way if ever they ask."`
+			label sentence
+			`	A third senator speaks up, a woman whose name card identifies her as Sara Ortega, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services. Councilman, please remain."`
+			`	You are led back out to the lobby, where you wait for hours. Finally, JJ comes and leads you to  Senator Ortega's office. `
+			`	The senator gestures for you both to sit down and pours you a drink. "Captain <last>, you've put us in a bind. News of your actions is spreading like wildfire, and much of the public is firmly behind you and your actions. Many of us personally feel great admiration for your actions on Greenrock. Obviously, we can't imprison someone who has freed so many slaves and destroyed such a serious threat. From a practical standpoint, we are almost certain to need you and your ship in the coming battles with the Republic.`
+			`	Senator Ortega sips her drink, then continues. "However, we are a government, and governments work on the rule of law. Your actions were morally correct, but legally, you've made a hell of a mess for us. Our government is still working out basic principles and the right balance of power. We cannot just allow ship captains in our forces to simply do as they please. Therefore you must have some kind of public reprimand and punishment.`
+			`	"Officially, we have decided that you will be suspended for your disobedience without pay. Separately, and rather unofficially, we have adjusted our clandestine operations budget to pay you a regular retainer. We will need your services again, and we hope you will come quickly when we contact you."`
+			`	JJ walks you out of the Senate building. "I'm so sorry for the way I treated you back on Longjump," he says. "When we thought a pirate armada was ready to strike, we were willing to do anything to break them up, even if it meant making a very public example of you." His face reddens. "It took the shock of seeing my sister again to make me see sense. I hope you'll forgive me for it someday." You shake hands, and feel him pass you a folded was of paper. Then JJ is gone, striding off to another committee meeting. 
+			`Back onboard <ship>, you open the paper. It reads,`
+			`	<first>,`
+			`	`
```

Comment:
[When I said above and below the letter](https://github.com/endless-sky/endless-sky/pull/2140#discussion_r100917917), I did not mean around the body of the letter. I meant above the greeting and below the signature.

I was apparently wrong to choose [Danforth's message](https://github.com/endless-sky/endless-sky/blob/master/data/free%20worlds%20middle.txt#L1300) as an example.